### ISSUE TITLE
Is there a way to disable autocomplete?

### ISSUE THREAD

**Issue by notcome** [b123400/purescript-ide-sublime#9]
Since we cannot make it faster as in #6.

**Comment by b123400** [comment]
Currently no, I should add an option in the next version. At the mean time try setting `auto_complete_timeout` to a small number?

**Comment by notcome** [comment]
Mine is set to 1 but still ran into scenario where it froze for more than ten seconds. However, I currently do not have time to figure out a minimal reproducing environment. The issue seems to occur after I use the `purs-loader` with Webpack (I used `pulp`) before. There is an option of using `pscIde: true` for fast reloading, which might or might not be an issue.

So far I am fine without `auto_complete` so I disabled it by setting the wrong path. By the way there seems to be another issue with the plugin that it cannot discover my binary on `/usr/local/bin`. I will provide a bit more info regarding my setup when I got more time.
